app/src/main/java/com/concordium/wallet/util/Extensions.kt

```diff
@@ -34,3 +35,17 @@ fun <T : Serializable?> Bundle.getSerializableFromBundle(key: String, m_class: C
         this.getSerializable(key) as T
     }
 }
+
+/**
+ * Allows to safely parse [BigInteger] from [String] with default value (0 by default)
+ * @return parsed value or [defaultValue] if it can't be parsed
+ */
+fun String?.toBigInteger(defaultValue: BigInteger = BigInteger.ZERO): BigInteger =
+    try {
+        if (this.isNullOrBlank())
```

Comment:
we can omit the `the` keyword since we are already in the scope of the extension function here so we would have.
 ```suggestion
        if (isNullOrBlank())
```